flakes/grep-dir.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -u
+set -o pipefail
+
+IFS=$'\n';
```

Comment:
What is it for?

Reply:
It is variable that sets how to split input in foreach. By default it is set to any whitespace, and in that scenario filenames with spaces in it are splited into few iterations of loop. When I set it to \n, whole lines are passed to each iteration.